packages/components/src/components/vertical-tab/vertical-tab-header.jsx

```diff
@@ -1,9 +1,8 @@
 import classNames  from 'classnames';
-import {
-    Counter,
-    Icon }         from '@deriv/components';
 import React       from 'react';
 import { NavLink } from 'react-router-dom';
+import Counter     from 'Components/counter';
+import Icon        from 'Components/icon';
 
 const HeaderIcon = ({ icon, is_active }) => (
     <Icon
```

Comment:
Please prefix all classnames in this file with `dc-` (with updates to their respective stylesheet).